### PR TITLE
Add translations for email messages

### PR DESCRIPTION
**Pull request by asaunier**
Related to https://github.com/c2corg/v6_api/issues/406

### CODE REVIEW

File: c2corg_api/emails/i18n/de/password_change_body [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Guten Tag
```
Comment (tsauerwein) [inline c2corg_api/emails/i18n/de/password_change_body]: Guten Tag,

um Ihr Kennwort zu ändern, klicken Sie bitte auf folgenden Link: %s

File: c2corg_api/emails/i18n/de/registration_body [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Guten Tag
```
Comment (tsauerwein) [inline c2corg_api/emails/i18n/de/registration_body]: Guten Tag,

um Ihre Registrierung abzuschließen, klicken Sie bitte auf folgenden Link: %s

**Comment by asaunier** [comment]
@tsauerwein I have integrated your suggestions, thanks.

Any other suggestions anyone? OK for merging?

**Comment by tsauerwein** [comment]
> Any other suggestions anyone? OK for merging?

The German and English translation looks ok for me, but I don't know for the other languages.

**Comment by asaunier** [comment]
OK thanks. No big deal anyway, it can be fixed later if needed.
I will merge when Travis is OK.

**Pull request merged**